<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="" lang="">
<head>
  <title>Test Results Report - actiTIME - chrome Smoke Test</title>
  <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
  <meta name="description" content="TestNG unit test results." />
  <link href="reportng.css" rel="stylesheet" type="text/css" />
    <script type="text/javascript" src="reportng.js"></script>
</head>
<body>
<h1>actiTIME - chrome Smoke Test</h1>
<p>
  Test duration: 82.721s
</p>



  <table class="resultsTable">
    <tr><th colspan="3" class="header failed">Failed Tests</th></tr>
          <tr>
        <td colspan="3" class="group">com.actitime.tests.web.CreateUser</td>
      </tr>
            
<tr>
  <td class="method">
                  createUser
      </td>
  <td class="duration">
    13.391s
  </td>
  <td class="result">
        
    
            <div class="testOutput">
                        UserAddition<a href=file:/C:/Users/anuj_d/Desktop/anuj/Github_Actitime/ActiTimeProject/actiTime-test-framework/Resources/ReportNGReports/Tue_Mar_28_17_45_31_IST_2017/%20UserAddition_Tue_Mar_28_17_45_31_IST_2017.png> click to open screenshot of UserAddition</a><br>
                                <br>
                  </div>
    

        <a href="javascript:toggleElement('exception-0', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: User having username: Vishal_Obd already exists!</b></a><br />
    <div class="stackTrace" id="exception-0">
              org.testng.Assert.fail(Assert.java:94)<br />
              com.actitime.tests.web.CreateUser.createUser(CreateUser.java:26)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br />
              java.lang.reflect.Method.invoke(Method.java:497)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:643)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:782)<br />
              org.testng.TestRunner.run(TestRunner.java:632)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:366)<br />
              org.testng.SuiteRunner.access$000(SuiteRunner.java:39)<br />
              org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)<br />
              org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)<br />
              java.util.concurrent.FutureTask.run(FutureTask.java:266)<br />
              java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)<br />
              java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)<br />
              java.lang.Thread.run(Thread.java:745)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.actitime.tests.web.DeleteUser</td>
      </tr>
            
<tr>
  <td class="method">
                  deleteUser
      </td>
  <td class="duration">
    14.850s
  </td>
  <td class="result">
        
    
            <div class="testOutput">
                        DeleteExistingUser<a href=file:/C:/Users/anuj_d/Desktop/anuj/Github_Actitime/ActiTimeProject/actiTime-test-framework/Resources/ReportNGReports/Tue_Mar_28_17_45_31_IST_2017/%20DeleteExistingUser_Tue_Mar_28_17_45_31_IST_2017.png> click to open screenshot of DeleteExistingUser</a><br>
                                <br>
                  </div>
    

        <a href="javascript:toggleElement('exception-1', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: AGC, PatientA : No Such User Found!</b></a><br />
    <div class="stackTrace" id="exception-1">
              org.testng.Assert.fail(Assert.java:94)<br />
              com.actitime.tests.web.DeleteUser.deleteUser(DeleteUser.java:30)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br />
              java.lang.reflect.Method.invoke(Method.java:497)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:643)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:782)<br />
              org.testng.TestRunner.run(TestRunner.java:632)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:366)<br />
              org.testng.SuiteRunner.access$000(SuiteRunner.java:39)<br />
              org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)<br />
              org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)<br />
              java.util.concurrent.FutureTask.run(FutureTask.java:266)<br />
              java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)<br />
              java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)<br />
              java.lang.Thread.run(Thread.java:745)<br />
                </div>
      </td>
</tr>
          <tr>
        <td colspan="3" class="group">com.actitime.tests.web.VerifyUser</td>
      </tr>
            
<tr>
  <td class="method">
                  verifyExistingUser
      </td>
  <td class="duration">
    15.583s
  </td>
  <td class="result">
        
    
            <div class="testOutput">
                        VerifyExistingUser<a href=file:/C:/Users/anuj_d/Desktop/anuj/Github_Actitime/ActiTimeProject/actiTime-test-framework/Resources/ReportNGReports/Tue_Mar_28_17_45_31_IST_2017/%20VerifyExistingUser_Tue_Mar_28_17_45_31_IST_2017.png> click to open screenshot of VerifyExistingUser</a><br>
                                <br>
                  </div>
    

        <a href="javascript:toggleElement('exception-2', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: AGC, PatientA : No Such User Found!</b></a><br />
    <div class="stackTrace" id="exception-2">
              org.testng.Assert.fail(Assert.java:94)<br />
              com.actitime.tests.web.VerifyUser.verifyExistingUser(VerifyUser.java:30)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br />
              java.lang.reflect.Method.invoke(Method.java:497)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:643)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:782)<br />
              org.testng.TestRunner.run(TestRunner.java:632)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:366)<br />
              org.testng.SuiteRunner.access$000(SuiteRunner.java:39)<br />
              org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)<br />
              org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)<br />
              java.util.concurrent.FutureTask.run(FutureTask.java:266)<br />
              java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)<br />
              java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)<br />
              java.lang.Thread.run(Thread.java:745)<br />
                </div>
      </td>
</tr>
      </table>


  <table class="resultsTable">
    <tr><th colspan="3" class="header passed">Passed Tests</th></tr>
          <tr>
        <td colspan="3" class="group">com.actitime.tests.web.Login</td>
      </tr>
            
<tr>
  <td class="method">
                  signIntoApplication
      </td>
  <td class="duration">
    16.353s
  </td>
  <td class="result">
        
    
            <div class="testOutput">
                        SignIntoApplication <a href=file:/C:/Users/anuj_d/Desktop/anuj/Github_Actitime/ActiTimeProject/actiTime-test-framework/Resources/ReportNGReports/Tue_Mar_28_17_45_31_IST_2017/%20SignIntoApplication%20_Tue_Mar_28_17_45_31_IST_2017.png> click to open screenshot of SignIntoApplication </a><br>
                                <br>
                  </div>
    

      </td>
</tr>
      </table>
  
</body>
</html>
